### PR TITLE
Renormalise hyperparam space, tune expansion, learning rate, capacity and nonlinearity

### PR DESCRIPTION
**Pull request by Marko-Shevchuk**

### CODE REVIEW

**Comment by Marko-Shevchuk** [comment]
Insufficient
